phys/module_sf_noahdrv.F

```diff
@@ -1776,6 +1781,19 @@ SUBROUTINE LSMINIT(VEGFRA,SNOW,SNOWC,SNOWH,CANWAT,SMSTAV,    &
 ! <--GAC
 
    IF(.not.restart)THEN
+   IF (irr_ph.NE.0)THEN
```

Comment:
This is where we need #if (EM_CORE==)

Reply:
Ok. I am making the changes also on the phy_init argument list so they work only with EM_CORE==1